Add monorepo move notice (#592)
netlify/edge-bundler: README.md

> # **Important Notice**
>

> > This repository was moved into the mono repository of [github.com/netlify/build](https://github.com/netlify/build) The
> > package name and the versions are preserved!

Reviewer: ```suggestion
> [!IMPORTANT]  
> This repository was moved into [the Netlify Build monorepo](https://github.com/netlify/build).
```
